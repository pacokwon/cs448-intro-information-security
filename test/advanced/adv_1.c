int main()
{
    int x = 0;
    int t = 10;

    if(x >= 10)
    {
        int y = 100 / x;
        int w = x / 0;
        t = 10 - x;
    }
    else if(x <= -10)
    {
        int y = 100 / x;

        t = 10 + x - x + x;
    }
    else if(x >= 0)
    {
        int y = 100 / x;
        t = t * x;
    }
    else
    {
        int y = 100 / x;
        t = t / y;
    }

    t = x / t;
}
