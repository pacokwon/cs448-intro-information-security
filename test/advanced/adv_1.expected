Divide-by-zero detected at line 20 and col 21
